kalyke/client.py

```diff
@@ -121,7 +121,8 @@ async def _send_message(self, registration_id, alert, identifier=None, expiratio
         headers = {
             'apns-expiration': str(expiration_time),
             'apns-priority': str(priority),
-            'apns-topic': topic
+            'apns-topic': topic,
+            'apns-push-type': push_type
```

Comment:
```suggestion
            'apns-push-type': push_type,
```

I also forgot to put a comma after the last element.